source/guide/3.2/guide-policies.md

```diff
@@ -106,33 +183,24 @@ Built-in policies are not special in any way - they use the same API any other c
 Advanced users may wish to write custom policies.
 To learn how to write a custom policy, refer to the [policies authoring guide](guide-authoring-policies.html).
 
-<!---
 # Auto Healing
 
 Auto healing is the process of automatically identifying when a certain component of your system is failing and fixing the system state without any user intervention.
 
-Cloudify comes with built-in support for auto healing different components.
+Cloudify comes with a built-in support for auto healing different components.
 
-There are several things that need to be configured to work with auto healing. This guide will walk you through them, and explain the limitations the mechanism currently has.
+There are several things that need to be configured for auto healing to work. This guide will walk you through them and explain the mechanism's current limitations.
 
 In short, the process involves:
 
-* Configuring monitoring on compute nodes that should have auto healing.
+* Configuring monitoring on compute nodes that will be subject to autohealing.
 * Adding the `autoheal` workflow to the blueprint.
 * Configuring the `groups` section with appropriate policy types and triggers.
 
-{%warning title=Limitations%}
-Make sure you don't miss the [Limitations](#limitations) section.
-{%endwarning%}
-
 ## Monitoring
 
 Add monitoring to compute nodes that require auto healing.
 
-At the moment, due to a limitation of existing policies, it is required that exactly one metric will be published by the monitored nodes.
-
-Another limitation requires that only compute nodes will have auto heal configured for them. Otherwise, when a certain compute node instance fails, node instances that are contained in it are also likely to fail, which in turn will cause the auto heal workflow to be triggered multiple times.
```

Comment:
I think it's only partly true - we can have a policy that, for instace, monitors if some process is working (using ps) and if it's not, we can heal the whole VM containing it. We only need to have an appropriate collector for this purpose. What we are completely unable to do is heal the node exclusively..